spec/system/admin/enterprises/index_spec.rb

```diff
@@ -39,8 +39,7 @@
       let!(:d_manager) { create(:user, enterprise_limit: 1) }
 
       before do
-        d_manager.enterprise_roles.build(enterprise: d).save
-        expect(d.owner).to_not eq d_manager
+        d.users << d_manager
```

Comment:
Enterprise owners are always users, so I guess this assignment would fail if somehow the manager was already the owner.

Reply:
Yes, it does fail: `Validation failed: That role is already present.`